pkg/sentry/socket/externalstack/stack.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package externalstack
+
+import (
+	"gvisor.dev/gvisor/pkg/sentry/inet"
+	"gvisor.dev/gvisor/pkg/sentry/stack"
+)
+
+type ExternalStack struct {
```

Comment:
These specific fields were initially implemented for optimizing TLDK in specific scenarios. After further consideration, however, we concluded that these fields might not be useful for a general external netstack. Therefore, for this initial version, we think it might be appropriate to remove these fields and keep ExternalStack as an implementation of inet.Stack.

Thank you for bringing this up and helping us make a more informed decision. We are happy to discuss more on making a proper definition of ExternalStack struct.

Reply:
I'm fine leaving them -- thanks for the explanations.